gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/BigQueryImplTest.java

```diff
@@ -39,7 +39,6 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Lists;
-import com.google.gcloud.AuthCredentials;
```

Comment:
testListJobsWithSelectedFields test has a `System.out.println` leftover. do you mind to remove it?